app/components/osf-navbar/home-links/template.hbs

```diff
@@ -1,34 +1,27 @@
 {{#if session.isAuthenticated}}
-    {{#if currentUser.user.canViewReviews}}
```

Comment:
This was removed because Abram realized he put it in by mistake when we were chatting about this PR, so I just pulled it here rather than making a new ticket.

Reply:
Oh, there actually IS a [ticket for this](https://openscience.atlassian.net/browse/EOSF-1047).